testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -4,7 +4,7 @@
 Feature: Synchronize products in the products page of the Setup Wizard
 
   Scenario: Let the products page appear
-    Given I am on the Admin page
+    Given I am authorized for the "Admin" section
     # Order matters here, refresh first
     When I refresh SCC
     And I follow "SUSE Products" in the content area
```

Comment:
Here I am not sure if it would work.

`I am authorized for the "Admin" section` just log you in as an admin user, but you end up on the normal start page.
Can you "follow SUSE Products" if you are on this page?

There are a few more tests like this.

Reply:
Good catch! Fixed.